ts/src/program/namespace/account.ts

```diff
@@ -201,21 +201,28 @@ export class AccountClient<
   async all(
     filters?: Buffer | GetProgramAccountsFilter[]
   ): Promise<ProgramAccount<T>[]> {
+    const filter: { offset?: number; bytes?: string; dataSize?: number } =
```

Comment:
Is it always the case that offset  and bytes are required to be together and dataSize is required to be alone?

If not, these changes would currently break filters that require all three fields.

Reply:
https://github.com/solana-labs/solana/blob/7e7785fc192b2d1b53eaf8f3d27c87c05b1d589c/web3.js/src/connection.ts#L2086

`offset` and `bytes` are required to be together, `dataSize` can either be alone or supplied with both `offset` and `bytes`.

Both of the `if` checks will pass if all three fields have been specified. Maybe I'm missing something?

Reply:
Nope. Thanks for the clarification!